extensions/amp-selector/1.0/component.js

```diff
@@ -372,18 +373,15 @@ export function Option({
     [customOnKeyDown, trySelect]
   );
 
-  const isSelected =
-    /** @type {!Array} */ (selected).includes(option) && !disabled;
-  const statusStyle =
-    disabled || selectorDisabled
-      ? CSS.DISABLED
-      : isSelected
-      ? selectorMultiple
-        ? CSS.MULTI_SELECTED
-        : CSS.SELECTED
-      : CSS.OPTION;
+  const isSelected = /** @type {!Array} */ (selected).includes(option);
   const optionProps = {
     ...rest,
+    className: objstr({
+      [classes.option]: true,
+      [classes.selected]: isSelected && !selectorMultiple,
```

Comment:
I kind of prefer this way, but just calling out that "selected" styles were only set when not disabled before. So just confirming this change is intended.

Reply:
Yes, thanks for calling this out explicitly. I'll add it to the PR description as well. Also preferred this way and intended to do this, hence why the relevant test cases were modified as well. This is more consistent with AMP layer this way.